src/parseSerialize/schemas.ts

```diff
@@ -347,21 +343,20 @@ export const transferSchemaV1: TSchema = {
   ],
 }
 
-export const transferSchemaV2: TSchema = {
+export const sponsorSchemaV1: TSchema = {
   type: 'object',
   schema: [
```

Comment:
Not sure if the order matters (if you use this order for actually producing the signature), if it does, timestamp and fee should be the other way around.